packages/api/scripts/cli.js

```diff
@@ -80,4 +84,59 @@ prog
       process.exit(1)
     }
   })
+  .command('db')
+  .describe('Database scripts')
+  .option('--reset', 'Reset db before running SQL.', false)
+  .action(async (opts) => {
+    const client = new Client({
+      user: process.env.PG_USER,
+      host: process.env.PG_HOST,
+      database: process.env.PG_DATABASE,
+      password: process.env.PG_PASSWORD,
+      // @ts-ignore
+      port: process.env.PG_PORT,
+    })
+    await client.connect()
+    const tables = fs.readFileSync(path.join(__dirname, '../db/tables.sql'), {
+      encoding: 'utf-8',
+    })
+    const functions = fs.readFileSync(
+      path.join(__dirname, '../db/functions.sql'),
+      {
+        encoding: 'utf-8',
+      }
+    )
+    const reset = fs.readFileSync(path.join(__dirname, '../db/reset.sql'), {
+      encoding: 'utf-8',
+    })
+    const cargo = fs.readFileSync(path.join(__dirname, '../db/cargo.sql'), {
+      encoding: 'utf-8',
+    })
+
+    if (opts.reset) {
+      await client.query(reset)
+    }
+    await client.query(tables)
+    await client.query(functions)
+
+    await client.query(
+      `
+CREATE EXTENSION IF NOT EXISTS postgres_fdw;
+
+DROP SERVER IF EXISTS dag_cargo_server CASCADE;
+
+CREATE SERVER dag_cargo_server
+  FOREIGN DATA WRAPPER postgres_fdw
+  OPTIONS (host '${process.env.DAG_CARGO_HOST}', dbname '${process.env.DAG_CARGO_DATABASE}', fetch_size '200000');
+
+CREATE USER MAPPING FOR current_user
+  SERVER dag_cargo_server
+  OPTIONS (user '${process.env.DAG_CARGO_USER}', password '${process.env.DAG_CARGO_PASSWORD}');
+    
+    `
```

Comment:
Please can this just be part of the `cargo.sql` file and just do some simple variable substitution (e.g. `$DAG_CARGO_HOST`)? If we need to change this I'd like to not have to hunt around for this part of the SQL. We also get syntax highlighting and linting in the editor if it's in a `.sql` file.

Reply:
imo its useful to do it like this, because it keeps the sql files runnable. 
i would do this change later when we have a proper sql migration system that support psql variable like sqitch.